main/index.js

```diff
@@ -29,6 +29,7 @@ app.on('ready', async () => {
   app.dock.hide();
 
   // Ensure all plugins are up to date
+  plugins.prune().catch(() => {});
   plugins.upgrade().catch(() => {});
```

Comment:
I don't think we should run both these commands at the same time. They might conflict.

Reply:
Hmm, it was doing it `onTransitionEnd`. Maybe make the transition longer? Or just wait a bit after the transition?